src/main/java/org/broadinstitute/consent/http/service/UserService.java

```diff
@@ -91,7 +91,8 @@ public User updateUserFieldsById(UserUpdateFields userUpdateFields, Integer user
                 userPropertyDAO.deletePropertiesByUserAndKey(userProps);
                 userPropertyDAO.insertAll(userProps);
             }
-            // Handle Roles.
+
+            // Handle Roles; must be admin to update roles.
```

Comment:
nit: no longer relevant in this case, the service class should handle roles regardless at this point.